packages/suite-desktop/src-electron/electron.ts

```diff
@@ -346,7 +349,11 @@ app.on('browser-window-focus', (_event, win) => {
 
 ipcMain.on('bridge/start', async (_event, devMode?: boolean) => {
     try {
-        await runBridgeProcess(devMode);
+        if (devMode) {
+            await bridge.startDev(forceBridge);
+        } else {
+            await bridge.start(forceBridge);
```

Comment:
starting bridge from a user request should be always forced regardless to flag, no?